acquia_cms.profile

```diff
@@ -38,6 +38,7 @@ function acquia_cms_install_tasks_alter(array &$tasks) {
   // This code helps capture time right when the drupal bootstrap happens.
   // The pre start function calls another method that performs the actual logic.
   $tasks['install_bootstrap_full']['function'] = 'acquia_cms_pre_start';
+  $tasks['install_finished']['function'] = 'acquia_cms_install_ui_kit_finished';
```

Comment:
@panshulK This is causing issue. We don't need this. Right ?
cc: @chandan-singh7929

Reply:
@vishalkhode1  The issue with this function has been resolved in this PR itself. We need the function back. This was accidently removed in @chandan-singh7929 's PR for 9.2.0 upgrade fix.